src/opnsense/mvc/app/controllers/OPNsense/Diagnostics/IpsecController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ *    Copyright (C) 2022 Manuel Faux <mfaux@conf.at>
+ *
+ *    All rights reserved.
+ *
+ *    Redistribution and use in source and binary forms, with or without
+ *    modification, are permitted provided that the following conditions are met:
+ *
+ *    1. Redistributions of source code must retain the above copyright notice,
+ *       this list of conditions and the following disclaimer.
+ *
+ *    2. Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *
+ *    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ *    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ *    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ *    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *    POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+namespace OPNsense\Diagnostics;
+
+use OPNsense\Base\IndexController;
+use OPNsense\Core\ACL;
+
+/**
+ * Class IpsecController
+ * @package OPNsense\Diagnostics
+ */
+class IpsecController extends IndexController
+{
+    /**
+     * IPsec status view
+     */
+    public function statusAction()
+    {
+        $acl = new ACL();
+        $this->view->allow_connection = $acl->isPageAccessible($this->session->get("Username"), "/api/diagnostics/ipsec/searchConnection");
```

Comment:
@8191 I think we better merge the ACL's into a single one as suggested earlier, there likely aren't may use-cases where people only should have access to part of this information and it removes complexity.

Reply:
@AdSchellevis Is there any straight-forward migration for such kind of ACL modifications? Or shall we just drop the unneeded and one needs to re-assign ACLs after in upgrade in case the dropped ones were used?

I suggest keeping "page-status-ipsec" and "page-status-ipsec-edit", while dropping "page-status-ipsec-sad" and "-spd".

Reply:
@8191 there's no migration for it, but given the relevancy of these acl's I would go for your suggestion to keep `page-status-ipsec` and be done with it.